### PR TITLE
Add starting files to this repo

### PR DESCRIPTION
**Pull request by RayStick**
Hi all, 

Thanks @Rainiefantasy and @eirini-zormpa for your feedback so far! I incorporated your specific and general feedback into this PR but let me know if there is something I missed.

This PR brings in the _starting content_ for this repository. 

All files are complete except _3-synthetic-datasets-inventory.md_ however I've put in [an issue](https://github.com/aim-rsf/Synthetic-Data/issues/16) to work on that later. The template of the inventory file is there, just not the content. 

**How to review the content:**

- Please feel free to look at the whole completed branch here to get a feel for the final formatting and content I am proposing: https://github.com/aim-rsf/Synthetic-Data/tree/add-starting-files-v2
- You can also see the 'Files changed' tab in this PR but that's less useful for the whole overview considering these are starting files, but feel free to comment on anything that way, using the PR functionalities (comments on specific files and lines for example). 

Let me know if you have specific edits, or you think anything big is missing at this stage.

If you are happy with it, please approve the PR via the Files Changed tab (with or without comments) but do not press merge.

Thanks 😸 💻 🧠

### CODE REVIEW

File: 3-synthetic-datasets-inventory.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+|CPRD cardiovascular disease synthetic dataset [^1] | CPRD Aurum database[^2] |  |    |   |                       |   | | |
+|CPRD COVID-19 symptoms and risk factors synthetic dataset[^1] | CPRD Aurum database[^2] |             |    |   |                       |   | | |
+|CPRD[^1]  Aurum sample dataset | CPRD Aurum database[^2] |             |    |   |                       |   | | |
+|UK Biobank Synthetic Dataset[^3] | UK Biobank[^4] |             |[Tofu](https://github.com/spiros/tofu) Python library  |   |                       |   | ||
```
Comment (eirini-zormpa) [inline 3-synthetic-datasets-inventory.md]: is this definitely what was used for this specific synthetic dataset of Biobank? It's just that the [methods](https://biobank.ndph.ox.ac.uk/~bbdatan/UKB_Synthetic_Dataset_Methods_Summary.pdf) for the creation don't mention it and the authors of those methods and the authors of the tofu package have no overlap!

File: illustrations/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Licenses for images/illustrations used within this repository 
+
+These images were created by Scriberia for The Turing Way community and are used under a CC-BY license:
+- DataArePeople.jpg
+- sensitive-data.jpg
+- utility-vs-privacy.jpg
+
+These images were created by Scriberia for AI for multiple long-term conditions: Research Support Facility and are used under a CC-BY license:
+- Using-Artificial-Intelligence-for-MLTC.jpg
```
Comment (eirini-zormpa) [inline illustrations/README.md:9]:
```suggestion
- Using-Artificial-Intelligence-for-MLTC.jpg

## References
- AIM RSF, & Scriberia. (2023). Illustrations for AI for multiple long-term conditions: Research Support Facility. Zenodo. https://doi.org/10.5281/zenodo.7919267
- The Turing Way Community, & Scriberia. (2023). Illustrations from The Turing Way: Shared under CC-BY 4.0 for reuse. Zenodo. https://doi.org/10.5281/zenodo.7587336
```

**Review by eirini-zormpa — APPROVED** [review]
I've added a couple of comments, but it otherwise looks good to me :rocket:

File: 3-synthetic-datasets-inventory.md (hunk 1 above)
Comment (RayStick) [reply 3-synthetic-datasets-inventory.md]: Good question ... I did not look into it very much. I will double check before merging. Thanks for flagging!

File: illustrations/README.md (hunk 2 above)
Comment (RayStick) [reply illustrations/README.md:9]: Accepted your commit - thanks!

**Pull request merged**